app/[slug]/(base)/usage/[[...metric]]/layout.tsx

```diff
@@ -25,12 +25,12 @@ export default async function RootLayout({
 
 	return (
 		<div className='space-y-2'>
-			<ChartsLinks
+			{/* <ChartsLinks
 				teamSlug={params.slug}
 				route={route}
-			/>
+			/> */}
 			<div className='space-y-5'>
-				<UsageCharts route={route} />
+				{/* <UsageCharts route={route} /> */}
```

Comment:
Should this code be uncommented?

Reply:
Yeah, charts isn't supported now since the usage schema is very different